public/locales/en/airdrop.json

```diff
@@ -24,7 +24,7 @@
         "text": "Tari Universe would like to use analytics to improve your experience and reward you with gems for mining."
     },
     "claimModalTitle": "EARN REWARDS FOR MINING ON TESTNET",
-    "claimModalText": "Earn gems by mining and referring friends. The more gems you earn, the larger your Tari airdrop. Have a referral code? Enter it below and earn <strong>{{gems}} bonus gems!</strong>",
+    "claimModalText": "Earn gems by mining and referring friends. The more gems you earn, the larger your Tari airdrop. Have a referral code? Enter it below and double your bonus from <strong>5,000 gems</strong> to <strong>10,000 gems!</strong>",
```

Comment:
Rather pass these values in, otherwise translations will not have them

Reply:
@stringhandler done!